marimbabot_audio/README.md

```diff
@@ -0,0 +1,26 @@
+# Sound2Note
+
+## 1. Introduction 
+This is a RosNode to translate the sound signal to western notation, based on the base version from Michael. 
+## 2. Installation
+### 2.1 Declaration
+This installation instruction has been currently verified only at the lab compute. 
+
+### 2.2 Installation of python virtual environment
+1. Come to your project position, the build the python virtualenv: `python -m venv env_sound2note --system-site-packages && source env_sound2note/bin/activate`
```

Comment:
Currently we don't use an venv for the project. Is it really needed?

Reply:
yes for now. If you worry about something, I can try to install the python packages in the local environment.

Reply:
I would prefer having all deps in the requirements.txt. If somebody wants to use a venv (it is usefull, I agree with that) they can use it and install the requirements.txt in it. Otherwise, it's a bit painful to integrate everything into a single launch file if different parts of the system use different venv and packaging methods.

Reply:
I think, I have integrated the necessary  dependencies into that `requirements.txt` file. So far, the visual part and audio part use the same `requirements.txt` , since you already merged the visual parts before.